rotkehlchen/chain/ethereum/modules/eth2/decoder.py

```diff
@@ -193,13 +200,26 @@ def _decode_eth2_consolidation_request(self, context: DecoderContext) -> Decodin
         source_public_key = Eth2PubKey(bytes_to_hexstr(context.tx_log.data[20:68]))
         target_public_key = Eth2PubKey(bytes_to_hexstr(context.tx_log.data[68:116]))
         if source_public_key == target_public_key:
-            target = self._get_indexes_or_public_keys(public_keys=[target_public_key])[0]
-            info_event.notes = f'Request to convert validator {target} into an accumulating validator'  # noqa: E501
+            if (indexes := self._get_known_validator_indexes(
+                context=context,
+                public_keys=[target_public_key],
+            )) is None:
+                return DEFAULT_DECODING_OUTPUT
+
+            info_event.extra_data = {'validator_index': (validator_index := indexes[0])}
+            info_event.notes = f'Request to convert validator {validator_index} into an accumulating validator'  # noqa: E501
+            info_event.event_subtype = HistoryEventSubType.UPDATE
```

Comment:
So this will get the "Update project" label? It may be a bit misleading. Wonder how it will look like in the UI now.